src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -1244,22 +1179,7 @@ private void LoadMembers()
                     this.CreateProperties(methodHandleToSymbol, nonFieldMembers);
                     this.CreateEvents(privateFieldNameToSymbols, methodHandleToSymbol, nonFieldMembers);
 
-                    foreach (PEFieldSymbol field in fieldMembers)
-                    {
-                        if ((object)field.AssociatedSymbol == null)
-                        {
-                            members.Add(field);
-                        }
-                        else
-                        {
-                            // As for source symbols, our public API presents the fiction that all
-                            // operations are performed on the event, rather than on the backing field.  
-                            // The backing field is not accessible through the API.  As an additional 
-                            // bonus, lookup is easier when the names don't collide.
-                            Debug.Assert(field.AssociatedSymbol.Kind == SymbolKind.Event);
-                        }
-                    }
-
+                    members.AddRange(fieldMembers);
```

Comment:
> members.AddRange(fieldMembers); [](start = 19, length = 32)

Does this also affects fields that back auto-properties?

Reply:
Fields that back auto-properties are already among the `GetMembers<FieldSymbol>(this.GetMembers(), SymbolKind.Field, offset: 0)` list along with ordinary fields.

Reply:
According to the original Debug.Assert, `field.AssociatedSymbol` is either null or its `Kind == SymbolKind.Event`. That means that property backing fields are never in `fieldMembers` at this point. I'm not sure why.